test/Conversion/TorchToLinalg/elementwise.mlir

```diff
@@ -102,3 +102,20 @@ func.func @elementwise_sinh(%arg0: !torch.vtensor<[3],f32>) -> !torch.vtensor<[3
   %0 = torch.aten.sinh %arg0 : !torch.vtensor<[3],f32> -> !torch.vtensor<[3],f32>
   return %0 : !torch.vtensor<[3],f32>
 }
+
+// -----
+
+// CHECK-LABEL:    func.func @elementwise_clamp
+// CHECK:            linalg.generic
+// CHECK:            arith.trunci
+// CHECK:            arith.cmpi slt
+// CHECK:            arith.select
+// CHECK:            arith.trunci
+// CHECK:            arith.cmpi sgt
+// CHECK:            arith.select
+func.func @elementwise_clamp(%arg0: !torch.vtensor<[1,3,8,8],si8>) -> !torch.vtensor<[1,3,8,8],si8> {
```

Comment:
For these type of changes, we use e2e tests rather than lit tests. See: https://github.com/llvm/torch-mlir/blob/main/docs/architecture.md#dos-and-donts-for-unit-vs-end-to-end-testing

Tests should be added here: https://github.com/llvm/torch-mlir/blob/1778314620b796de7a7aba61f00396cecbd29a0b/projects/pt1/python/torch_mlir_e2e_test/test_suite/elementwise.py#L860

Reply:
Updated and removed the old tests.